packages/bot-web-ui/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "../../tsconfig.json",
     "compilerOptions": {
+        "experimentalDecorators": true,
```

Comment:
Experimental support for decorators is a feature that is subject to change in a future release. Set the 'experimentalDecorators' option in 'tsconfig' or 'jsconfig' to remove this warning.